drivers/gles3/storage/texture_storage.h

```diff
@@ -404,8 +404,8 @@ class TextureStorage : public RendererTextureStorage {
 	RID_Owner<CanvasTexture, true> canvas_texture_owner;
 
 	/* Texture API */
-
-	mutable RID_Owner<Texture> texture_owner;
+	//textures can be created from threads, so this RID_Owner is thread safe
```

Comment:
```suggestion
	// Textures can be created from threads, so this RID_Owner is thread safe.
```

Reply:
should I change this in the vulkan driver as well then? I just copied this from there ^^